e2e/interchaintestv8/cosmos_relayer_test.go

```diff
@@ -488,5 +515,32 @@ func (s *CosmosRelayerTestSuite) ICS20TimeoutPacketTest(ctx context.Context, num
 			s.Require().NotNil(resp.Balance)
 			s.Require().Equal(testvalues.InitialBalance, resp.Balance.Amount.Int64())
 		}))
+
+		s.Require().True(s.Run("Verify balances on Chain B", func() {
+			resp, err := e2esuite.GRPCQuery[banktypes.QueryBalanceResponse](ctx, s.SimdB, &banktypes.QueryBalanceRequest{
+				Address: simdBUser.FormattedAddress(),
+				Denom:   s.SimdB.Config().Denom,
+			})
+			s.Require().NoError(err)
+			s.Require().NotNil(resp.Balance)
+			s.Require().Equal(originalBalance, resp.Balance)
+		}))
+	}), 10*time.Minute)
+
+	s.Require().True(s.Run("Constructing relay packet after timeout should fail", func() {
+		resp, err := s.RelayerClient.RelayByTx(context.Background(), &relayertypes.RelayByTxRequest{
+			SrcChain:       s.SimdA.Config().ChainID,
+			DstChain:       s.SimdB.Config().ChainID,
+			TimeoutTxIds:   txHashes,
+			TargetClientId: ibctesting.FirstClientID,
+		})
+		s.Require().Error(err)
+		s.Require().Nil(resp)
 	}))
+
+	s.Require().True(s.Run("Receiving packets on Chain B after timeout should fail", func() {
+		resp, err := s.BroadcastSdkTxBodyGetResult(ctx, s.SimdB, s.SimdBSubmitter, 2_000_000, txBodyBz)
+		s.Require().Error(err)
+		s.Require().Nil(resp)
```

Comment:
Is the error possible to verify here as well?

Reply:
Here it works correctly(error returned). Only the other part with comment note is NoError.

Reply:
I think we wanted to assert the actual error but I'm fine with this